Packs/Akamai_SIEM/README.md

```diff
@@ -0,0 +1,43 @@
+<~XSIAM>
+ 
+## Overview
+A Web Application Firewall (WAF) is a security filter that protects against HTTP-based attacks by inspecting traffic before it reaches your application.
+ 
+## This pack includes:
+ 
+Data normalization capabilities:
+  * Rules for Parsing and Modeling logs that are ingested via the Native Collector on Cortex XSIAM.
+  * The ingested Akamai WAF SIEM logs can be queried in XQL Search using the *`akamai_waf_raw`* dataset.
+ 
+***
+ 
+## Data Collection
+ 
+### Akamai WAF side
+
+1. Go to `WEB & DATA CENTER SECURITY` > `Security Configuration` > choose your configuration > `Advanced settings` > Enable SIEM integration.
+2. [Open Control panel](https://control.akamai.com/) and login with admin account.
+3. Open `identity and access management` menu.
+4. Create a user with assigned roles `Manage SIEM` or make sure the admin has rights to manage SIEM.
```

Comment:
Create a user with `Manage SIEM` permissions or make sure the admin has permission to manage the SIEM.